docs/under the hood/garbage-collection.md

```diff
@@ -36,60 +38,17 @@ That is, it is hard-deleted only after the retention period has ended for all re
 Example GC rules for a repository:
 ```json
 {
-  "default_retention_days": 14,
+  "default_retention_days": 21,
```

Comment:
Why this change?

Also, this deletes #3554 and its two fixes #3561, #3562.  I assume that this is by accident!  Note that it is hard to perform a huge change on documentation when it is concurrently being updated by others.

Reply:
I've been editing files locally but using Github Desktop, I made sure to pull updates regularly and right before pushing changes. Perhaps something didn't work. I guess the best way to go forward would be to reject my change - I'm happy to go over the file later on (for editing/proofreading)